set.mm

```diff
@@ -369106,6 +369106,95 @@ varying definitions (some start from 0, others start from 1), but
        renamer theorem like ~ cbval . The section
        ~ mmtheorems32.html#mm3146s also describes the
        metatheorem that underlies this.
+       </p>
+
+       <p>
+       Standard Metamath verifiers do not distinguish between axioms and
+       definitions (both are $a statements).
+       In practice, we require that definitions (1) not be creative
+       (a definition should not allow an expression
+       that previously qualified as a wff but was not provable
+       to become provable) and be eliminable
+       (there should exist an algorithmic method for converting any
+       expression using the definition into
+       a logically equivalent expression that previously qualified as a wff).
+       To ensure this, we have additional rules on almost all definitions
+       ($a statements with a label that does not begin with ax-).
+       These additional rules are not applied in a few cases where they
+       are too strict ( ~ df-bi , ~ df-clab , ~ df-cleq , and ~ df-clel );
+       see those definitions for more information.
+       These additional rules for definitions are checked by at least
+       mmj2's definition check (see
+       <A HREF="https://github.com/digama0/mmj2/blob/master/"
+       >mmj2 master</a> file mmj2jar/macros/definitionCheck.js).
```

Comment:
This could be a direct link? Is it because you want to stay under 80 chars?

Reply:
Right.  It was originally a direct link but then it can't commit because Travis requires all lines to be under 80 chars.

Reply:
@nmegill Thoughts? I could have sworn this has come up before.

Reply:
@digama0 - it's a really long URL, I don't remember there being a "nice" way to deal with this.